### DISCUSSION TITLE
Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://verify.walletconnect.com') does not match the recipient window's origin

### DISCUSSION THREAD

**Discussion by robidity** [WalletConnect/walletconnect-monorepo#3431] [Developer Support] [answered]
Hi, I configured my project on your dashboard, but I'm still getting this error in production when trying to connect my wallet.
I have verified my domain and added to allowed domains (and waited for more than 15 minutes):
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/102060249/6e62f6c5-eeff-4898-876c-d49a58d80bd3)

My DNS record:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/102060249/61ab2405-84fe-47d2-9943-2b758850422d)

And checked on MXToolBox:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/102060249/4d270e1d-5638-4c7f-86b4-060dff7fdb88)

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/102060249/3cbdaa5e-b092-4340-975b-73c8f6a6b0a0)

It seems everything is fine, but still getting the error, may I have any help please?

**Comment by glitch-txs** [comment]
hi, do you have a public repo with a minimal repro? I think this is not related to the cloud verification, you don't need to be verified to use WalletConnect :)

**Reply by robidity** [reply, marked as answer]
Hey, I was using WalletConnect through ThirdWeb library and it was my mistake.
You were right, so now it's working fine.
Thank you anyway! ;)
